src/main/java/frc/robot/subsystems/Ramp.java

```diff
@@ -32,15 +34,10 @@ private void configureMotor() {
         neoPidMotor.setSmartMotionAllowedClosedLoopError(Constants.RAMP_ERROR_RANGE);
         neoPidMotor.setMaxAccel(Constants.RAMP_MAX_RPM_ACCELERATION);
         neoPidMotor.getPidController().setP(Constants.RAMP_PID_P);
-        neoPidMotor.getPidController().setFF(Constants.RAMP_PID_FAR_FF);
+        neoPidMotor.getPidController().setFF(neoModerFF);
```

Comment:
If this is being set in the `teleopInit` and `autoInit` then why do it in construction too?

Reply:
Unknown. Leaving in place.,